flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/factories/TestValuesTableFactory.java

```diff
@@ -879,6 +890,7 @@ private TestValuesScanTableSourceWithoutProjectionPushDown(
                 DataType producedDataType,
                 ChangelogMode changelogMode,
                 boolean bounded,
+                boolean finite,
```

Comment:
Use the enum here as well? Also for bounded? Or is it due to the serialization?

Reply:
No it was to be consistent with bounded. I did not want to change the class too much. I'll do that though.